install_zutils.py

```diff
@@ -611,6 +611,9 @@ def main():
     parser.add_argument(
         "--pcg", action="store_true", default=False, help="Whether or not to install PCG"
     )
+    parser.add_argument(
+        "--pcgtag", action="store_true", default="v2.18.3", help="PCG repo tag to use"
```

Comment:
Parser will complain if you specify this as a flag and pass a value
```suggestion
        "--pcgtag", default="v2.18.3", help="PCG repo tag to use"
```

Reply:
removed